libs/SalesforceSDKCore/SalesforceSDKCore/Classes/PushNotification/PushNotificationManager.swift

```diff
@@ -224,6 +202,8 @@ public class PushNotificationManager: NSObject {
             return false
         }
         
+        let restClient = notificationRegister.client(for: user)
```

Comment:
sorry last one, could you also add a guard or something for the rest client and log an error if it's nil for some reason? I ran into a similar case recently and just want to make sure it can't fail silently. same for the similar block in the unregister method

Reply:
Yes.  I'll update that this morning.